audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/AudioRecorderViewModel.kt

```diff
@@ -13,7 +14,7 @@ abstract class AudioRecorderViewModel : ViewModel() {
     abstract fun isRecording(): Boolean
     abstract fun getCurrentSession(): LiveData<RecordingSession?>
 
-    abstract fun start(sessionId: String, output: Output)
+    abstract fun start(sessionId: Serializable, output: Output)
```

Comment:
We need to use `Seriablizable` instead of `Object`/`Any` here as the identifier needs to be able to be passed as an argument to an Android `Service`.